.github/workflows/self_hosted_e2e.yaml

```
@@ -110,13 +110,17 @@ jobs:
             fi
           fi
         shell: bash
+      - name: Set the test timeout - MacOS
+        if: matrix.os == 'macOS-latest'
```

```suggestion
        if: matrix.os == 'macos-latest'
```
Let's be consistent within file and have lowercase